docs/.vuepress/components/ScriptLoader.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+</template>
+
+<script>
+const decode = (base64data) => {
+  return decodeURI(base64data);
+};
+
+const useCodePreview = (code) => {
+  const scriptElement = document.createElement('script');
+
+  scriptElement.type = 'text/javascript';
+  scriptElement.innerHTML = code;
+
+  return [
+    (container) => { container.appendChild(scriptElement); },
+    () => { scriptElement.parentElement.removeChild(scriptElement); },
+  ];
+};
+
+export default {
+  name: 'ScriptLoader',
+  props: ['code'],
+  mounted() {
+    const [append, remove] = useCodePreview(decode(this.$props.code));
+
+    append(this.$el);
+    remove();
```

Comment:
Can someone explain to me how it works? We append and at the same time remove?

Reply:
The script is parsed, compiled, and then removed. The code still runs in the memory. 

But yeah in the context of the live debugging it'd be a better idea to remove the script in the Vue "destroy" event.

Reply:
Fixed within 2562370052e0d773e4142f51827036b2ff4d458e.

Reply:
As far as I remember, There was:
```js
this.unmount = remove
```